examples/docs_example/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@osdk/examples.docs.example",
+  "private": true,
+  "version": "0.0.1",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/palantir/osdk-ts.git"
+  },
+  "scripts": {
+    "build": "tsc && vite build",
+    "codegen": "rm -rf src/generatedNoCheck/* src/generatedNoCheck2/* && osdk unstable typescript generate --outDir src/generatedNoCheck2 --beta --ontologyPath ontology.json --version dev",
```

Comment:
You can probably just collapse this into src/generatedNoCheck (with --beta still)